src/os/windows/mod.rs

```diff
@@ -171,6 +171,40 @@ impl Library {
         ret
     }
 
+    /// Attempts to pin the module represented by the current `Library` into memory.
+    /// If successful, the module will remain in memory regardless of the refcount for this `Library`
```

Comment:
Empty line between summary line and the rest of doc comment.

```suggestion

    /// If successful, the module will remain in memory regardless of the refcount for this `Library`
```

Plus adding some text describing the relevant native APIs would be nice, so that this comes up in search when somebody pastes a `GET_MODULE_HANDLE_EX_FLAG_PIN` or something.